deploy/ansible/roles/iroha-cluster-deploy-node/templates/config.j2

```diff
@@ -0,0 +1,10 @@
+{
+  "block_store_path" : "/tmp/block_store/",
+  "torii_port" : {{ torii_port + item | int }},
+  "internal_port" : {{ internal_port + item | int }},
+  "pg_opt" : "host={{ postgresName }}_{{ key + item | int }}_1 port={{ postgresPort }} user={{ postgresUser }} password='{{ postgresPassword }}'",
```

Comment:
You could've defined `container_name` in docker-compose file to not guess container names by suffixing them with `_1`

Reply:
@bakhtin good point